tests/foreman/cli/test_setting.py

```diff
@@ -86,3 +86,129 @@ def test_negative_discover_host_with_invalid_prefix(self):
 
         :caseautomation: notautomated
         """
+
+    @stubbed
+    @tier1
+    def test_positive_update_email_delivery_method_smtp(self):
+        """Check Updating SMTP params through settings subcommand
+
+        :id: b26798e9-528c-4ed6-a09b-dc8e4668a8d7
+
+        :steps:
+            1. set "delivery_method" to smtp
+            2. set all smtp related properties:
+                2.1. smtp_address
+                2.2. smtp_authentication
+                2.3. smtp_domain
+                2.4. smtp_enable_starttls_auto
+                2.5. smtp_openssl_verify_mode
+                2.6. smtp_password
+                2.7. smtp_port
+                2.8. smtp_user_name
+
+        :expectedresults: SMTP properties are updated
+
+        :CaseImportance: Critical
+
+        :caseautomation: notautomated
+        """
+
+    @stubbed
+    @tier1
+    def test_positive_update_email_delivery_method_sendmail(self):
+        """Check Updating Sendmail params through settings subcommand
+
+        :id: 578de898-fde2-4957-b39a-9dd059f490bf
+
+        :steps:
+            1. set "delivery_method" to sendmail
+            2. set all sendmail related properties:
+                2.1. sendmail_arguments
+                2.2. sendmail_location
+
+        :expectedresults: Sendmail properties are updated
+
+        :CaseImportance: Critical
+
+        :caseautomation: notautomated
+        """
+
+    @stubbed
+    @tier1
+    def test_positive_update_email_reply_address(self):
+        """Check email reply address is updated
+
+        :id: cb0907d1-9cb6-45c4-b2bb-e2790ea55f16
+
+        :expectedresults: email_reply_address is updated
+
+        :caseautomation: notautomated
+        """
+
+    @stubbed
+    @tier1
+    def test_negative_update_email_reply_address(self):
+        """Check email reply address is not updated
+
+        :id: 2a2220c2-badf-47d5-ba3f-e6329930ab39
+
+        :steps: provide invalid email addresses
+
+        :expectedresults: email_reply_address is not updated
+
+        :caseautomation: notautomated
+        """
+
+    @stubbed
+    @tier1
+    def test_positive_update_email_subject_prefix(self):
+        """Check email subject prefix is updated
+
+        :id: c8e6b323-7b39-43d6-a9f1-5474f920bba2
+
```

Comment:
The test title is so straight forward that I feel no step description is need since I would end up just repeating the title on it. What you think? @rochacbruno same for you bellow comment ;)